### DISCUSSION TITLE
need help meteor

### DISCUSSION THREAD

**Discussion by andriux26** [jekhokie/raspberry-noaa-v2#419] [General]
hi noaa is decoding very nice but meteor not decoding i get meteor file obout 350-360mb but not image wats wrong 
meteor config 
meteor_m2_schedule: true
meteor_m2_sdr_device_id: 0
meteor_m2_freq_offset: -22
meteor_m2_enable_bias_tee: true
meteor_m2_gain: 48
meteor_m2_sun_min_elevation: 0
meteor_m2_sat_min_elevation: 30
meteor_m2_memory_threshold: 200

**Comment by RyLeeRyno** [comment]
Your frequency offset being set to -22 is culprit.  Try the default value of -6.
Also set the memory threshhold to 600
and lastly, gain:  0

basically restore the default values which have been edited to ranges outside of the recommended values.

**Reply by rustytravis** [reply]
Thank you for such a reply. I also am unable to decode meteor passes. Do I need a different METEOR_RECEIVER value? I'm currently using a RTL-SDR v3 dongle, but the settings.yml comments don't suggest any SDR devices that will work for the use of "gnuradio".

Some diagnostic evidence:

$ grep -i meteor  ~/.noaa-v2.conf
METEOR_AUDIO_OUTPUT=/srv/audio/meteor
METEOR_M2_SCHEDULE=true
METEOR_M2_SDR_DEVICE_ID=0
METEOR_M2_FREQ_OFFSET=-6
METEOR_M2_ENABLE_BIAS_TEE="-T"
METEOR_M2_GAIN=7.7
METEOR_M2_SUN_MIN_ELEV=-90
METEOR_M2_SAT_MIN_ELEV=30
METEOR_M2_MEMORY_TRESHOLD=600
FLIP_METEOR_IMG=true
METEOR_RECEIVER=rtl_fm

Email from the "at" job:

INFO : The system have enough space to store a Meteor pass on RAM
INFO : Free : 1287 ; Required : 600
INFO : Direction Southbound
INFO : Starting rtl_fm record
INFO : Demodulation in progress (QPSK)
INFO : Producing spectrogram
INFO : Producing polar graph of direction for pass
INFO : Free memory : 794 ; Total RAMFS usage : 355M
INFO : Deleting audio files
INFO : Decoding in progress (QPSK to BMP)
INFO : I got a successful METEOR-M-2-20210826-090852.dec file. Creating false color image
INFO : Rectifying image to adjust aspect ratio
INFO : Annotating images and creating thumbnails
rm: cannot remove '/srv/images/METEOR-M-2-20210826-090852-122.bmp': No such file or directory
rm: cannot remove '/srv/audio/meteor/METEOR-M-2-20210826-090852.bmp': No such file or directory
INFO : No image produced - not pushing anywhere
INFO : Total processing time: 00:35.58

**Reply by ve3elb** [reply]
There is an install bug issue with meteor in v1.7. I have fix this bug and others in the RaspiNOAA V2 disk image I compiled. You can download it at 

https://qsl.net/ve3elb/RaspiNOAA/

I know most like to compile their own but this image helps with some of the bug issues and I have added a few mods. Give it a try. Enjoy.

**Comment by dom-robinson** [comment]
i'd suggest meteor_m2_gain: 16.4 as a start. 

Never use '0' - the autogain rarely works well.

**Reply by RyLeeRyno** [reply]
I beg to differ.  ![image](https://user-images.githubusercontent.com/76919948/126980422-f6e6dcfd-8ebd-4c3d-abc4-2dce6cf5fa27.jpeg)

![image](https://user-images.githubusercontent.com/76919948/126980469-088ece16-c787-4a8d-8f95-41ab2852fab1.jpeg)

I've had better pass consistency with auto gain enabled.  I'm using a v dipole mounted one meter off the ground, lna, and rtl-sdr v3.

**Comment by andriux26** [comment]
1.6 version works very good but 1.7 not decode meteor noaa decoding very good   i use rtl-sdr v3 and raspberry 4

**Reply by ve3elb** [reply]
There is an install bug issue with meteor in v1.7. I have fix this bug and others in the RaspiNOAA V2 disk image I compiled. You can download it at 

https://qsl.net/ve3elb/RaspiNOAA/

I know most like to compile their own but this image helps with some of the bug issues and I have added a few mods. Give it a try. Enjoy.

**Comment by rustytravis** [comment]
On 8/27/21 2:56 AM, ve3elb wrote:
>
> There is an install bug issue with meteor in v1.7. I have fix this bug 
> and others in the RaspiNOAA V2 disk image I compiled. You can download 
> it at
>
> https://qsl.net/ve3elb/RaspiNOAA/ <https://qsl.net/ve3elb/RaspiNOAA/>
>
> I know most like to compile their own but this image helps with some 
> of the bug issues and I have added a few mods. Give it a try. Enjoy.
>

Corporate has allowed the install of raspberry-noaa-v2 on this 128 GB 
RPi which does many other things besides, but I'm not authorized to 
replace the entire Raspbian OS.

Is it possible to just get a replacement raspberry-noaa-v2 that includes 
the bug fixes, please?

**Comment by rustytravis** [comment]
Thank you, but no thank you. Installing a customized OS outside of 
distribution channels is how worms can get introduced into an OS by 
trusting individuals. We need an updated raspberry-noaa-v2, nothing 
more. The use of ansible and sudo should make your "bug issue" 
corrections transparent to the pi user.
